src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets

```diff
@@ -103,4 +103,10 @@ Copyright (c) .NET Foundation. All rights reserved.
       Include="$(MSBuildThisFileDirectory)..\codestyle\vb\Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.dll"
       IsImplicitlyDefined="true" />
   </ItemGroup>
+  
+  <!-- Build Properties Exposed to CodeStyle Analyzers -->
+  <ItemGroup Condition="'$(Language)' == 'C#' Or '$(Language)' == 'VB'">
```

Comment:
If this will only used for sync namespace analyzer, then `'$(Language)' == 'VB'` would be unnecessary.

Reply:
why can't sync namespaces work for VB? both languages have namespaces?